addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -4122,7 +4122,8 @@ export class OdooEditor extends EventTarget {
         }
         // Remove now empty links
         for (const link of element.querySelectorAll('a')) {
-            if (![...link.childNodes].some(isVisible) && !link.classList.length) {
+            if (![...link.childNodes].some(isVisible) && !link.classList.length
+                && !('t-out' in link.attributes)) {
```

Comment:
You can also includes here the `t-esc`  attribute since there are some other template that uses it to make sure we cover all cases. 
```suggestion
            if (![...link.childNodes].some(isVisible) && !link.classList.length
                && !['t-out', 't-esc'].some(attr => link.hasAttribute(attr)) {
```